src/simulator.jl

```diff
@@ -151,6 +151,9 @@ function Simulator(; map_name::String=DEFAULT_MAP_NAME, max_steps::Int=DEFAULT_M
     return sim
 end
 
+RandomMapSimulator(;kwargs...) = Simulator(;tiles = generate_random_track(), custom_map = true,
```

Comment:
Should there be `tiles` and `custom_map` kwargs? Because when `map_name="custom_map"` that is be implied.

Reply:
I prefer to think of the map_name not as the indicator of a random map. In essence I can have a custom_map.yaml that I might want to get parsed

Reply:
Fair enough. So when I have `custom_map=true` then should I pass `tiles` as well, or should they be generated randomly?